### PR TITLE
tr translated

### PR DESCRIPTION
**Pull request by AYakici**

### CODE REVIEW

**Comment by AYakici** [comment]
translated en to tr , Student 9 AyhanYakici

File: locale/tr_TR.UTF-8/LC_MESSAGES/palma.po [hunk 1]
```diff
@@ -0,0 +1,515 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PalMA 0.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-06-29 18:50+0200\n"
+"PO-Revision-Date: 2017-09-12 14:38+0200\n"
+"Last-Translator: Dennis Müller <dennis.mueller@bib.uni-mannheim.de>\n"
+"Language-Team: English <en@li.org>\n"
+"Language: en\n"
```
Comment (stweil) [inline locale/tr_TR.UTF-8/LC_MESSAGES/palma.po:15]: Please update these lines, too.

File: locale/tr_TR.UTF-8/LC_MESSAGES/palma.po [hunk 2]
```diff
@@ -0,0 +1,515 @@
...
+msgid "Remove this item"
+msgstr "Bu öğeyi sil"
+
+#: index.php:706
+msgid "Jump to start"
+msgstr "Başlamak için zıpla"
+
+#: index.php:707
+msgid "Page up"
+msgstr ""
```
Comment (stweil) [inline locale/tr_TR.UTF-8/LC_MESSAGES/palma.po:71]: Sayfa yukarı?

**Review by stweil — CHANGES_REQUESTED** [review]
Thanks a lot for adding a new language. Please see my other comments for the small parts which still need changes.

Please note that your contribution will be honored in the README [here](https://github.com/UB-Mannheim/PalMA/tree/master/locale) after your pull request was merged. Therefore your pull request should have your full real name as author and in the signature. That's currently not the case. Please try to fix that.

**Pull request closed without merging**